lib/rust/profiler/src/flame_graph.rs

```diff
@@ -0,0 +1,159 @@
+//! The `flame_graph` module contains functionality that allows the profiling framework to
```

Comment:
Remove the name of the module from the documentation. It makes the files way harder to rename.